libraries/core-react/src/components/Slider/Slider.tsx

```diff
@@ -122,7 +118,7 @@ const Wrapper = styled.div<WrapperProps>`
   &:hover:not([disabled]) {
     ${fakeTrackBgHover}
     &::after {
-      background: ${enabled.track.indicator.hover.color};
+      background: ${track.states.hover.background};
```

Comment:
This should probably be `indicator.states.hover.background`?

Reply:
Fixed 👍